### ISSUE TITLE
No way to set a theme color to the default foreground color

### ISSUE THREAD

**Issue by Nemo157** [lsd-rs/lsd#896]
`crossterm::style::Color` only supports the 8-bit ansi color codes, there is no way to make it emit code `39` which uses the default foreground color (which can be distinct from the normal and bright variants of both `white` and `black`).

As a workaround, adding support for parsing a string like `reset` to `Color::Reset` should give a similar result.

**Comment by Destroy666x** [comment]
Yes, please add support for `Reset` as it currently does not work: https://docs.rs/crossterm/latest/crossterm/style/enum.Color.html#variant.Reset

**Comment by Destroy666x** [comment]
Fixing above would likely be the easiest way to fix this from what I see, as this just uses the deserialization. Otherwise a workaround is needed.
